langchain/chains/openai_functions/extraction.py

```diff
@@ -32,11 +32,16 @@ def _get_extraction_function(entity_schema: dict) -> dict:
 
 
 _EXTRACTION_TEMPLATE = """Extract and save the relevant entities mentioned\
- in the following passage together with their properties.
+
```

Comment:
don't think we want a new line here, do we?

Reply:
Yeah, that new line was not necessary